week-2/implement/bmi.js

```diff
@@ -13,3 +13,9 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+function IBM(weightKG,heightM){
+    let ibm=weightKG/(heightM*heightM);
```

Comment:
this could be a const as it is only assigned once

Reply:
Alright. I will fix it.